### ISSUE TITLE
Support importing other class

### ISSUE THREAD

**Issue by m3m0r7** [php-java/php-java#17]

**Comment by chitoku-k** [comment]
- **PHP**
```
PHP 7.3.5 (cli) (built: May  2 2019 12:40:36) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.5, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.5, Copyright (c) 1999-2018, by Zend Technologies
```

- **JDK**
```
openjdk version "1.8.0_202"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_202-b08)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.202-b08, mixed mode)
```

- **PHPJava**
fcf7e88493d8dc6b90dba4a9bfb767dda7623197

Another class that belongs to the same package as `main` method is not imported as well. I made a reproducible repo for testing purpose: https://github.com/chitoku-k/php-java-test/commit/802b84d5156e760620b470face0be932efe0c7f3

Stack trace:

```
Fatal error: Uncaught Error: Function name must be a string in /Users/chitoku/Test/php-java-playground/vendor/php-java/php-java/src/Kernel/
Internal/InstanceDeferredLoader.php:40
Stack trace:
#0 /Users/chitoku/Test/php-java-playground/vendor/php-java/php-java/src/Kernel/Mnemonics/_invokespecial.php(66): PHPJava\Kernel\Internal\In
stanceDeferredLoader->instantiate()
#1 /Users/chitoku/Test/php-java-playground/vendor/php-java/php-java/src/Core/JVM/Invoker/Invokable.php(260): PHPJava\Kernel\Mnemonics\_invo
kespecial->execute()
#2 /Users/chitoku/Test/php-java-playground/test.php(12): PHPJava\Core\JVM\Invoker\StaticMethodInvoker->call('main', Array)
#3 {main}
  thrown in /Users/chitoku/Test/php-java-playground/vendor/php-java/php-java/src/Kernel/Internal/InstanceDeferredLoader.php on line 40
```

**Comment by chitoku-k** [comment]
Thanks. I confirmed that this was fixed in #138.
